snap-server/src/main/java/com/kyc/snap/crossword/CrosswordSpreadsheetWrapper.java

```diff
@@ -104,12 +105,18 @@ public void toSpreadsheet(Grid grid, Crossword crossword, CrosswordClues clues)
                     allCharsExpression,
                     references.get(p))));
         }
-
+        
+        spreadsheets.setFont(rowOffset, grid.getNumRows(), colOffset, grid.getNumCols(), MONOSPACE_FONT);
+        spreadsheets.setTextAlignment(rowOffset, grid.getNumRows(), colOffset, grid.getNumCols(), "CENTER", "MIDDLE");
         spreadsheets.setProtectedRange(rowOffset, grid.getNumRows(), colOffset, grid.getNumCols());
         for (int i = 0; i < clues.getSections().size(); i++) {
             int numRows = clues.getSections().get(i).getClues().size();
             if (numRows > 0)
                 spreadsheets.setProtectedRange(0, numRows, directionColumns.get(i), 2);
+            	// Ensure that any long clues are auto-wrapped
+            	spreadsheets.setWrapStrategy(0, numRows, directionColumns.get(i), 1, "WRAP");
+            	// Format columns for enumerations and user-input to monospace
+            	spreadsheets.setFont(0, numRows, directionColumns.get(i) + 1, 2, MONOSPACE_FONT);
```

Comment:
I really need a better shim class so this code path doesn't have to do like 20 external requests in series